src/envoy/http/mixer/control.cc

```diff
@@ -33,21 +33,30 @@ Control::Control(const Config& config, Upstream::ClusterManager& cm,
                  [this](::istio::mixerclient::Statistics* stat) -> bool {
                    return GetStats(stat);
                  }) {
+  if (!config_.config_pb()
+           .transport()
+           .attributes_for_mixer_proxy()
+           .attributes()
+           .empty()) {
+    config_.config_pb()
+        .transport()
+        .attributes_for_mixer_proxy()
+        .SerializeToString(&serialized_forward_attributes_);
+  }
```

Comment:
Can we define a function to share this code with tcp?   similar to Utils::CreateEnvironment()

Reply:
Done